src/components/adaptive-cards/Spacer/Spacer.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * Spacer helper component
+ *
+ * @param {object} props  React props passed to the component
+ * @param {string} props.spacing  The type of spacing
+ * @param {boolean} props.separator  Whether or not to display the separator
+ * @param {object} props.containerType  The type of the parent container
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+export default function Spacer({
+  className,
+  separator,
+  spacing,
+  style,
+}) {
+  const spacingType = spacing;
+  const isSeparator = separator;
+  const spacingClassName = `spacing-${String(spacingType).toLowerCase()}`;
+  const separatorClassName = 'separator';
+
+  const [cssClasses] = webexComponentClasses('ac-spacer', className, {
+    [spacingClassName]: !!spacingType,
+    [separatorClassName]: !!isSeparator,
```

Comment:
maybe put this class directly on the separator:
```js
{separator && <div class={sc('separator')} />}
```
the flex css rule can always go on the container.